5_databases/week2/task1_2.sql

```diff
@@ -0,0 +1,74 @@
+USE hyf_lesson1;
+
+--1. Add a task  with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (title, description, created, updated, due_date, status_id, user_id)
+VALUES ('Be good', 'Stop beating your husband', '2023-11-16 12:00:00', '2023-11-16 12:00:00', '2023-12-01 12:00:00', 1, 3);
+
+--2. Change the title of a task
+UPDATE task
+set title = "Sleep"
+where title = "Wash clothes";
+
+
+SELECT *
+from task;
+
+--3. Change a task due date
+UPDATE task
+SET due_date = 2017-10-15 
+WHERE title = 'Wash clothes';
+
+--4. Change a task status
+UPDATE status
```

Comment:
Here you update a status. You should update the status of the task instead

Reply:
Hej Adionditsak, the name is actually a status label:
 insert into status (id, name) values (1, 'Not started');